.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,45 @@
+Thanks for contributing to Ecosphères!
+
+_Fixes / Relates to / Supercedes / Superceded by / Depends on / Depended upon by_ #XXX
+
+### Here is the user or job story
+
+> User: _kind of user_.
+>
+> When I _situation_,
+> I want to _motivation_,
+> so I can _expected outcome_.
+
+### Here is the context or situation
+
+_Write here._
+
+### Here is the problem encountered by users
+
+_Write here._
+
+### Here is a proposal of how to solve the problem
+
+_Write here. Add UI/UX elements if you have them._
+
+#### Breaking changes
```

Comment:
Could we maybe collapse the "changes" section asking to highlight what changed? Unless this is used to generate a changelog?
My reasoning is we can look at the code and commit messages to see what changed in details, so having to list it here is redundant (redundant = more work, risk of inconsistencies, bias in review, ...).
BUT we do want highlights still: anything a reviewer will want to pay specific attention to, anything that's not obvious from code/commits, etc.

Reply:
Sure, the aim is to class changes semantically (breaking, minor, patch), more as a CHANGELOG.